sdks/java/io/file-based-io-tests/src/test/java/org/apache/beam/sdk/io/text/TextIOIT.java

```diff
@@ -68,59 +72,45 @@ public static void setup() throws ParseException {
     IOTestPipelineOptions options = TestPipeline.testingPipelineOptions()
         .as(IOTestPipelineOptions.class);
 
-    if (options.getLinesOfTextCount() != null) {
-      linesOfTextCount = options.getLinesOfTextCount();
-    }
-
+    linesOfTextCount = options.getNumberOfRecords();
```

Comment:
Sorry - seems that I forgot to change the variable name as well. I'll change it to _numberOfTextLines_ in the next code review iteration as we can be more specific here, I guess... Will this be ok?